tests/behat/features/links.feature

```diff
@@ -0,0 +1,47 @@
+@civictheme @civictheme_links
+Feature: Check that links have the correct classes.
+
+  @api @javascript
```

Comment:
this should not be a JS test as there is no JS processing involved

Reply:
is required to run And I wait for AJAX to finish
Alerts listing